src/test/java/org/apache/sling/jcr/resource/internal/JcrResourceListenerTest.java

```diff
@@ -269,7 +269,7 @@ public boolean matches(String path) {
                 session.save();
             }
             System.out.println("Events = " + events);
-            assertEquals("Received: " + events, 7, events.size());
+            assertEquals("Received: " + events, 6, events.size());
```

Comment:
Why does this unrelated test needs to be adjusted?

Reply:
See https://issues.apache.org/jira/browse/SLING-11063, therefore also in a dedicated commit https://github.com/apache/sling-org-apache-sling-jcr-resource/pull/19/commits/1cec8179d182cef5471fda8f1c0bff091347b3fd

Reply:
Ok, thanks - makes sense